From a2a702315cc4872ff3a6cac3f3ec20f06b863f36 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Mon, 9 Oct 2017 12:45:33 +0300
Subject: [PATCH 2023/2241] imvebu: dts: a80x0: a70x0: align node names in
 device trees

align node names in device trees by remove "0x" from PCIe nodes
The node names should be "unit@address" e.g. pci0@60000 usb@50000

Change-Id: I7c1d53497ac074f5b4bee3016cc71b96912c2a96
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/45072
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-3900-db-axis.dts          |  2 +-
 arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts            |  2 +-
 arch/arm64/boot/dts/marvell/armada-3900-db.dtsi              |  4 ++--
 arch/arm64/boot/dts/marvell/armada-7020-amc.dts              |  2 +-
 arch/arm64/boot/dts/marvell/armada-7020.dtsi                 |  6 +++---
 arch/arm64/boot/dts/marvell/armada-7040-db-B.dts             |  6 +++---
 arch/arm64/boot/dts/marvell/armada-7040-db-C.dts             |  6 +++---
 arch/arm64/boot/dts/marvell/armada-7040-db-D.dts             |  6 +++---
 arch/arm64/boot/dts/marvell/armada-7040-db-E.dts             |  6 +++---
 arch/arm64/boot/dts/marvell/armada-7040-db-all-pcie.dts      |  6 +++---
 arch/arm64/boot/dts/marvell/armada-7040-db-nand.dts          |  4 ++--
 arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts   |  2 +-
 arch/arm64/boot/dts/marvell/armada-7040-db.dts               |  2 +-
 arch/arm64/boot/dts/marvell/armada-7040-pcac.dts             |  2 +-
 arch/arm64/boot/dts/marvell/armada-7040-pd.dts               |  2 +-
 arch/arm64/boot/dts/marvell/armada-7040.dtsi                 |  6 +++---
 arch/arm64/boot/dts/marvell/armada-70x0-customer.dts         |  6 +++---
 arch/arm64/boot/dts/marvell/armada-8020.dtsi                 |  6 +++---
 arch/arm64/boot/dts/marvell/armada-8040-db-B.dts             | 10 +++++-----
 arch/arm64/boot/dts/marvell/armada-8040-db-C.dts             | 10 +++++-----
 arch/arm64/boot/dts/marvell/armada-8040-db-D.dts             | 10 +++++-----
 arch/arm64/boot/dts/marvell/armada-8040-db-E.dts             | 10 +++++-----
 arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts   | 10 +++++-----
 arch/arm64/boot/dts/marvell/armada-8040-db.dts               | 10 +++++-----
 .../arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts |  2 +-
 arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts            |  2 +-
 arch/arm64/boot/dts/marvell/armada-8040-ocp.dts              |  2 +-
 arch/arm64/boot/dts/marvell/armada-8040.dtsi                 | 12 ++++++------
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi                |  8 ++++----
 29 files changed, 81 insertions(+), 81 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db-axis.dts b/arch/arm64/boot/dts/marvell/armada-3900-db-axis.dts
index 8a9c372..2f47129 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db-axis.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db-axis.dts
@@ -66,7 +66,7 @@
 			};
 		};
 
-		pcie0@0x600000 {
+		pcie0@600000 {
 			phys = <&cpm_comphy 0 COMPHY_PCIE0>;
 			phy-names = "pcie-phy0";
 		};
diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts b/arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts
index f6c351c..b264064 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts
@@ -61,7 +61,7 @@
 			};
 		};
 
-		pcie0@0x600000 {
+		pcie0@600000 {
 			num-lanes = <2>;
 			phys = <&cpm_comphy 0 COMPHY_PCIE0
 				&cpm_comphy 1 COMPHY_PCIE0
diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
index 1f10478e..d67f712 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
@@ -184,12 +184,12 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 			ranges = <0x81000000 0 0xf7000000 0  0xf7000000 0 0x10000	/* downstream I/O */
 				  0x82000000 0 0xf6000000 0  0xf6000000 0 0xf00000>;	/* non-prefetchable memory */
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 			phys = <&cpm_comphy 5 COMPHY_PCIE2>;
 			phy-names = "pcie-phy0";
diff --git a/arch/arm64/boot/dts/marvell/armada-7020-amc.dts b/arch/arm64/boot/dts/marvell/armada-7020-amc.dts
index 8db97d0..8d2cf15 100644
--- a/arch/arm64/boot/dts/marvell/armada-7020-amc.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7020-amc.dts
@@ -159,7 +159,7 @@
 			};
 		};
 
-		pcie0@0x600000 {
+		pcie0@600000 {
 			/* overwrite the PCIe memory address since it request for 64MB */
 			ranges = <0x81000000 0 0xfa000000 0  0xfa000000 0 0x10000 /* downstream I/O */
 				  0x82000000 0 0xf6000000 0  0xf6000000 0 0x6000000>; /* non-prefetchable memory */
diff --git a/arch/arm64/boot/dts/marvell/armada-7020.dtsi b/arch/arm64/boot/dts/marvell/armada-7020.dtsi
index 12af0ff..da7ad15 100644
--- a/arch/arm64/boot/dts/marvell/armada-7020.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-7020.dtsi
@@ -136,13 +136,13 @@
 				status = "disabled";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			msi-parent = <&gic_v2m0>;
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			msi-parent = <&gic_v2m0>;
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			msi-parent = <&gic_v2m0>;
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts
index 197b365..0eadd99 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts
@@ -171,14 +171,14 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 			num-lanes = <4>;
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			status = "disabled";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts
index 9fc2b0d..37fe59f 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts
@@ -198,13 +198,13 @@
 			};
 		};
 
-		pcie0@0x600000{
+		pcie0@600000{
 			status="okay";
 		};
-		pcie1@0x620000{
+		pcie1@620000{
 			status = "disabled";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
index 54e19bf..e836c2a 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
@@ -202,13 +202,13 @@
 			};
 		};
 
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			status = "disabled";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
index f8811a6..d4edd89e 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
@@ -170,13 +170,13 @@
 			};
 		};
 
-		pcie0@0x640000 {
+		pcie0@640000 {
 			status = "okay";
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			status = "disabled";
 		};
-		pcie2@0x600000 {
+		pcie2@600000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-all-pcie.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-all-pcie.dts
index 3f440f2..f45de54 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-all-pcie.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-all-pcie.dts
@@ -153,15 +153,15 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 			num-lanes = <4>;
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			status = "okay";
 		};
 
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-nand.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-nand.dts
index 6fb20e4..032aee9 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-nand.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-nand.dts
@@ -178,10 +178,10 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
index c031eb4..9915c00 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
@@ -198,7 +198,7 @@
 			};
 		};
 
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dts b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
index 656e606..bd15daf 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
@@ -181,7 +181,7 @@
 			};
 		};
 
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-pcac.dts b/arch/arm64/boot/dts/marvell/armada-7040-pcac.dts
index f373578..feccefc 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-pcac.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-pcac.dts
@@ -227,7 +227,7 @@
 			status = "okay";
 		};
 
-		pcie-ep@0x600000 {
+		pcie-ep@600000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-pd.dts b/arch/arm64/boot/dts/marvell/armada-7040-pd.dts
index 5538d3a..f184e95 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-pd.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-pd.dts
@@ -270,7 +270,7 @@
 			};
 		};
 
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-7040.dtsi b/arch/arm64/boot/dts/marvell/armada-7040.dtsi
index 28db509..b708863 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-7040.dtsi
@@ -135,13 +135,13 @@
 				status = "disabled";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			msi-parent = <&gic_v2m0>;
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			msi-parent = <&gic_v2m0>;
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			msi-parent = <&gic_v2m0>;
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-70x0-customer.dts b/arch/arm64/boot/dts/marvell/armada-70x0-customer.dts
index 1ca61f4..f42d1d3 100644
--- a/arch/arm64/boot/dts/marvell/armada-70x0-customer.dts
+++ b/arch/arm64/boot/dts/marvell/armada-70x0-customer.dts
@@ -285,15 +285,15 @@
 				status = "disabled";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "disabled";
 			/* UPDATE */
 			num-lanes = <4>;
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			status = "disabled";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "disabled";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-8020.dtsi b/arch/arm64/boot/dts/marvell/armada-8020.dtsi
index 3079a31..db3109d 100644
--- a/arch/arm64/boot/dts/marvell/armada-8020.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-8020.dtsi
@@ -148,13 +148,13 @@
 				#stream-id-cells = <1>;
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			msi-parent = <&gic_v2m0>;
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			msi-parent = <&gic_v2m0>;
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			msi-parent = <&gic_v2m0>;
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
index edc3c69..4f6f688 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
@@ -148,11 +148,11 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 			num-lanes = <4>;
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
@@ -225,13 +225,13 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			status = "okay";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts
index 2f05419..4c0f56a 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts
@@ -163,10 +163,10 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
@@ -253,13 +253,13 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			status = "okay";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts
index 5bbb84d..2bfba68 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts
@@ -164,10 +164,10 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
@@ -230,13 +230,13 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			status = "okay";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
index e269451..7a84173 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
@@ -166,10 +166,10 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
@@ -245,13 +245,13 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			status = "okay";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
index 223ccde..2d8fb2d 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
@@ -173,10 +173,10 @@
 				status = "disabled";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
@@ -258,13 +258,13 @@
 				pinctrl-0 = <&cps_spi1_pins>;
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			status = "okay";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db.dts b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
index 1fde910..071d5c2 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
@@ -164,10 +164,10 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
@@ -243,13 +243,13 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			status = "okay";
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			status = "okay";
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts
index da3491d..edf6130 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts
@@ -201,7 +201,7 @@
 				};
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			num-lanes = <4>;
 			status = "okay";
 			reset-gpios = <&cpm_gpio1 20 GPIO_ACTIVE_HIGH>;
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
index d819a70..450da44 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
@@ -164,7 +164,7 @@
 				};
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			num-lanes = <4>;
 			status = "okay";
 			reset-gpios = <&cpm_gpio1 20 GPIO_ACTIVE_HIGH>;
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-ocp.dts b/arch/arm64/boot/dts/marvell/armada-8040-ocp.dts
index e8418f9..985f876 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-ocp.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-ocp.dts
@@ -146,7 +146,7 @@
 			status = "okay";
 		};
 
-		pcie-ep@0x600000 {
+		pcie-ep@600000 {
 			status = "okay";
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040.dtsi b/arch/arm64/boot/dts/marvell/armada-8040.dtsi
index e6235e2..0b7e57f 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-8040.dtsi
@@ -161,13 +161,13 @@
 				#stream-id-cells = <1>;
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			msi-parent = <&gic_v2m0>;
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			msi-parent = <&gic_v2m0>;
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			msi-parent = <&gic_v2m0>;
 		};
 	};
@@ -207,13 +207,13 @@
 				status = "okay";
 			};
 		};
-		pcie0@0x600000 {
+		pcie0@600000 {
 			msi-parent = <&gic_v2m0>;
 		};
-		pcie1@0x620000 {
+		pcie1@620000 {
 			msi-parent = <&gic_v2m0>;
 		};
-		pcie2@0x640000 {
+		pcie2@640000 {
 			msi-parent = <&gic_v2m0>;
 		};
 	};
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index 88b4d3c..d0f057c 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -523,7 +523,7 @@ config-space {
 	};
 };
 
-CP110_LABEL(pcie_ep): pcie-ep@0x600000 {
+CP110_LABEL(pcie_ep): pcie-ep@600000 {
 	compatible = "marvell,armada8k-pcie-ep", "snps,dw-pcie";
 	reg = <0 CP110_PCIE_EP_REG0_BASE 0 0x4000>,
 		  <0 CP110_PCIE_EP_REG1_BASE 0 0x80000>,
@@ -533,7 +533,7 @@ CP110_LABEL(pcie_ep): pcie-ep@0x600000 {
 	status = "disabled";
 };
 
-CP110_LABEL(pcie0): pcie0@0x600000 {
+CP110_LABEL(pcie0): pcie0@600000 {
 	compatible = "marvell,armada8k-pcie", "snps,dw-pcie";
 	reg = <0 CP110_PCIEx_REG0_BASE(0) 0 0x10000>,
 	      <0 CP110_PCIEx_REG1_BASE(0) 0 0x80000>;	/* Last 512KB of mem space */
@@ -558,7 +558,7 @@ CP110_LABEL(pcie0): pcie0@0x600000 {
 	status = "disabled";
 };
 
-CP110_LABEL(pcie1): pcie1@0x620000 {
+CP110_LABEL(pcie1): pcie1@620000 {
 	compatible = "marvell,armada8k-pcie", "snps,dw-pcie";
 	reg = <0 CP110_PCIEx_REG0_BASE(1) 0 0x10000>,
 	      <0 CP110_PCIEx_REG1_BASE(1) 0 0x80000>;	/* Last 512KB of mem space */
@@ -584,7 +584,7 @@ CP110_LABEL(pcie1): pcie1@0x620000 {
 	status = "disabled";
 };
 
-CP110_LABEL(pcie2): pcie2@0x640000 {
+CP110_LABEL(pcie2): pcie2@640000 {
 	compatible = "marvell,armada8k-pcie", "snps,dw-pcie";
 	reg = <0 CP110_PCIEx_REG0_BASE(2) 0 0x10000>,
 	      <0 CP110_PCIEx_REG1_BASE(2) 0 0x80000>;	/* Last 64KB of mem space */
-- 
2.7.4

